target/linux/ramips/dts/mt7628an_kroks.dtsi

```diff
@@ -0,0 +1,42 @@
+#include "mt7628an.dtsi"
+
+&ethernet {
+	mtd-mac-address = <&factory 0x4>;
```

Comment:
We use nvmem-cells now. With this, MAC address will be random during test. Did you test with master at all?

Reply:
No, will do ASAP when I get the access to the device.